tests/e2e/scripts/init.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+# Script to install pre-requisites on vsphere node to run e2e test
+
+export PATH=$PATH:/usr/local/go/bin/:/usr/local/bin
+echo 'Installing Kubectl'
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
+sudo mv kubectl /usr/local/bin/ && \
```

Comment:
It usually is not a good idea to add `sudo` commands in a shellscript, but instead call the script itself with `sudo` if such access is needed.

Reply:
Calling the script with sudo results in permission denied error for non-root user to run the tests.

Reply:
Perhaps then splitting the installation parts that need sudo into one script and running the tests into another? The mix with having sudo on some lines in a script is not very appealing.